docs/docsite/rst/conf.py

```diff
@@ -193,6 +210,19 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'Poseidodoc'
 
+# Configuration for sphinx-notfound-pages
+# with no 'notfound_template' and no 'notfound_context' set,
+# the extension builds 404.rst into a location-agnostic 404 page
+#
+# default is `en` - using this for the sub-site:
+notfound_default_language = "ansible"
```

Comment:
What is this?

Reply:
Got it from Ansible itself, a guess is default code-block when language is not mentioned.